src/Elmish.WPF/BindingInternal.fs

```diff
@@ -38,14 +38,14 @@ type internal TwoWayData<'model, 'msg, 'a> =
     Set: 'a -> 'model -> 'msg }
 
 
-type internal CmdData<'model, 'msg> = {
+type internal CmdData<'model, 'msg, 'cmd> = {
```

Comment:
Continuing the discussion started by @the-not-mad-psychologist in https://github.com/elmish/Elmish.WPF/pull/471#discussion_r888928206.

I am also not convinced that this is necessary.  Can you try removing it?  I might need to look at this in Visual Studio.

Reply:
Ok I removed it. It's actually unnecessary, especially at this point in the transformation. I don't think it's ever necessary either, though, because we always have this in the context of a containing type that *does* have this type information.